src/cli/discord.ts

```diff
@@ -88,7 +104,7 @@ async function getMainWindow (browser: Browser): Promise<Page> {
 
 export default async function (apiPort: number): Promise<Readonly<DiscordInstance>> {
   const discordExecutable = await findDiscord()
-  if (!discordExecutable) throw new Error('Cannot find Discord')
+  if (!discordExecutable) throw new Error('Cannot find Discord, If you have Discord installed on a custom dir you need to edit it yourself.')
```

Comment:
Given that this is intended to be installed as a npm lib, editing the library's code is a really bad idea. Leave the error untouched, as a better mitigation will be implemented later.